client/src/components/TaskListView/TaskListView.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import styles from "./TaskListView.module.css";
+import TaskCard from "../TaskCard/TaskCard";
+import { PropTypes } from "prop-types";
+
+const TaskListView = ({ tasks, taskEdittable }) => {
```

Comment:
I would rename this to `edittable` since we're already in the context of a TaskList

Reply:
Renamed to `edittable`